### ISSUE TITLE
Upgrade vscode library

### ISSUE THREAD

**Issue by TheColorRed** [HvyIndustries/crane#55]
We probably should upgrade the vscode library

> npm WARN deprecated vscode@0.10.7: Please update to version 0.11.x for improved typings and test support

**Comment by nevadascout** [comment]
Updated in 6db7798d6957dd726061bc33b2b11c8f59bddcb0

**Comment by nevadascout** [comment]
I've done the update, but there seem to be some issues with typescript not finding a reference to `vscode` anywhere.

Needs further testing

**Comment by TheColorRed** [comment]
The following needs to be added to the `scripts` block in the `package.json`. https://github.com/Microsoft/vscode/issues/2810#issuecomment-182209917

```
"postinstall": "node ./node_modules/vscode/bin/install"
```

After doing so though, I am still getting error to the package...

**Comment by TheColorRed** [comment]
Here is a config that seems to be working:

Set the minimum version:
```json
    "engines": {
        "vscode": "^0.10.x"
    }
```

Set the actual version:
```json
    "devDependencies": {
        "typescript": "^1.6.2",
        "vscode": "^0.11.0"
    }
```

I then ran the following:

```
[sudo] npm install --unsafe-perm
```

Afterwards I had to restart vscode for the errors to go away.

**Comment by nevadascout** [comment]
Resolved in a5d36088093880dc01f4a5a470f967ade55af8dc
